content/asset-tracking/index.textile

```diff
@@ -0,0 +1,165 @@
+    - Resolution policy#resolution-policy
+    - See also#see-also
+---
+
+The Ably Asset Tracking solution provides a way to track the location of multiple assets in realtime. An asset is any object that moves geographically, and whose position needs to be monitored. The Ably Asset Tracking solution is powered by the Ably realtime network, the Ably Asset Tracking SDKs, and the Mapbox Navigation SDK with location enhancement.
+
+The Ably Asset Tracking solution provides two SDKs:
+
+* Publishing SDK - for embedding in apps on the asset to be tracked.
+* Subscribing SDK - for embedding in apps that want to observe the asset being tracked.
+
+As Ably is used as the underlying transport, you have direct access to your data and can use "Ably Integrations":https://ably.com/integrations for a wide range of applications, in addition to direct realtime subscriptions. Examples include: 
+
+* Passing data to a third-party service.
+* Persistence of data to a database for later retrieval.
+* Monitoring.
+
+As the asset being tracked moves, it publishes its position through the app to an Ably channel. Any subscribers to the channel can then receive positional information, which can be displayed on a detailed map.
+
+The following diagram provides an overview of an asset tracking use case:
+
+<a href="/images/screenshots/asset-tracking/overview.png" target="_blank">
+  <img src="/images/screenshots/asset-tracking/overview.png" style="width: 100%" alt="Overview">
+</a>
+
+In a typical asset tracking scenario the key participants are:
+
+- asset (trackable) := Item that the customer wants to track such as driver, vehicle, container, and so on. This also known as a @Trackable@ in the SDKs.
+- driver := Person driving the asset physically. Could also be a machine moving the asset under software direction, such as in the case of a delivery drone.
+- customer := Entity that uses the Ably Asset Tracking SDKs to build their product to solve the asset tracking problem for their users.
+- end user := Person that uses the product built by the customer and uses the asset tracking capabilities.
+
+Each asset being tracked must have a @trackableId@, and this must be globally unique within an Ably asset tracking account.
+
+The Ably Asset Tracking SDKs also allow for various battery optimization scenarios, so frequency of updates can be modified based on various parameters such as distance to destination, subscriber preference, map zoom level, and current battery levels.
+
+h2(#battery-optimization). Battery optimization scenarios
+
+One of the biggest challenges in asset tracking is combining high resolution of updates with low battery consumption on the asset tracking device. For example, in a delivery tracking use case you may want your users to have a smooth and realtime visualization of where their delivery is on the map, but at the same time sending locations from the driver's device frequently will rapidly drain its battery.
```

Comment:
I think this focusses too much on battery optimisation as the reason for controlling resolution. It's not just that - it might be bandwidth costs, for example.

The heading needs to change I think to be about "dynamic resolution" or "dynamic update frequency".

I suggest something like:

The Asset Tracking SDKs allow the frequency of publication of location updates to be controlled based on a range of parameters, both by the publisher and the subscriber. Having this control, to vary the frequency of updates, and sensor(s) used to obtain location information, is important to be able to control the resource usage of the application. In many mobile applications, for example, conservation of the battery life of the courier's device is paramount; high frequency and high resolution location updates that are broadcast unconditionally are not compatible with the battery constraints for mobile delivery riders, for example. The Asset Tracking SDKs aim to provide a sufficient level of control to be able to optimise the multiple parameters that influence bandwidth and energy usage.

Reply:
I agree with the change of wording, but I also think that battery usage was highlighted as one of the primary value drivers for customers. We probably need to address it in some way in our docs eventually, as people will come to them with question in mind "How do I optimise battery usage by using Ably Asset Tracking?" as this is one of the things we will advertise. Therefore, I think docs should give an answer to this, that is easy to find. 

Should we potentially add Battery Optimisations as a separate topic (page) in the Asset Tracking section? Or should it be something like a Tutorial? 

@tbedford / @paddybyers  what do you think?

Reply:
When this doc is expanded there's going to have to be more extensive discussion relating to resolution policies, and we should have a whole section on how resolution policy should be tuned to optimise battery life, and what parameters to pass to the default policy to optimise battery life. I just don't think it should be the high-level statement of what dynamic resolution is for.